src/planscape/planscape/settings.py

```diff
@@ -96,6 +97,14 @@
         "PORT": 5432,
         "USER": "planscape",
     },
+    "clearcreek": {
```

Comment:
should we remove the default DB?

Reply:
I've removed the "clearcreek" one instead.  It's the better solution.